### PR TITLE
DLS-8898 update pack at business address to match formatting used in …

### PR DESCRIPTION
**Pull request by GinnyThomas**
…registrations - inset text

### CODE REVIEW

File: app/viewmodels/AddressFormattingHelper.scala [hunk 1]
```diff
@@ -18,8 +18,20 @@ package viewmodels
...
 object AddressFormattingHelper {
+
+  def formatBusinessAddress(ukAddress: UkAddress, tradingName: Option[String]): HtmlContent = {
+    HtmlContent(tradingName.fold("")(tradingName => tradingName + "<br/>") +
+      ukAddress.lines.map(line => if (line.isEmpty) {
+        ""
+      } else {
+        HtmlFormat.escape(line).toString() + "<br/>"
+      }).mkString +
+      s"""<span class="nowrap" style="white-space: nowrap;">${ukAddress.postCode}</span>""")
```
Comment (drewmiley) [inline app/viewmodels/AddressFormattingHelper.scala]: Why do we need the nowrap? I feel like there would be a cleaner way to do this - depending on the exact reasoning why

File: app/viewmodels/AddressFormattingHelper.scala [hunk 2]
```diff
@@ -18,8 +18,20 @@ package viewmodels
 
 import models.backend.UkAddress
 import play.twirl.api.{Html, HtmlFormat}
+import uk.gov.hmrc.govukfrontend.views.Aliases.HtmlContent
 
 object AddressFormattingHelper {
+
+  def formatBusinessAddress(ukAddress: UkAddress, tradingName: Option[String]): HtmlContent = {
+    HtmlContent(tradingName.fold("")(tradingName => tradingName + "<br/>") +
```
Comment (drewmiley) [inline app/viewmodels/AddressFormattingHelper.scala]: Will think if there is a slightly cleaner way to do this

File: app/viewmodels/AddressFormattingHelper.scala (hunk 1 above)
Comment (GinnyThomas) [reply app/viewmodels/AddressFormattingHelper.scala]: I was following the pattern used in registration, however, as post code is on it's own line in this usage, it should not be needed.  Removed and re-pushed code.

File: app/viewmodels/AddressFormattingHelper.scala (hunk 2 above)
Comment (drewmiley) [reply app/viewmodels/AddressFormattingHelper.scala]: Filter ukAddresses by empty lines and then join using "<br/>"? The bit in the middle might be a little cleaner then

File: app/viewmodels/AddressFormattingHelper.scala (hunk 1 above)
Comment (GinnyThomas) [reply app/viewmodels/AddressFormattingHelper.scala]: updated

File: app/views/PackAtBusinessAddressView.scala.html [hunk 3]
```diff
@@ -36,16 +35,18 @@
...
         }
 
-       <h1 class="govuk-heading-l">@messages("packAtBusinessAddress.title") </h1>
-
-        <p class="govuk-body-m"> <span class="govuk-visually-hidden">@messages("packAtBusinessAddress.nameText")</span> @businessName</p>
-        <p class="govuk-body-m"> <span class="govuk-visually-hidden">@messages("packAtBusinessAddress.addressText")</span> @businessAddress.lines.head, @businessAddress.lines.last, @businessAddress.postCode</p>
+        <h1 class="govuk-heading-l">@messages("packAtBusinessAddress.title") </h1>
 
+        @govukInsetText(InsetText(
+            id = Some("utrNo"),
```
Comment (drewmiley) [inline app/views/PackAtBusinessAddressView.scala.html]: Should this id be changed as it doesn't reflect what is displayed by the insetText?

**Review by drewmiley — COMMENTED** [review]
One comment - rest looks good

**Review by drewmiley — APPROVED** [review]

**Pull request merged**